src/content/quickstart/openshift/_index.md

```diff
@@ -137,10 +141,44 @@ subctl join --kubeconfig cluster-a/auth/kubeconfig broker-info.subm --cluster-id
 subctl join --kubeconfig cluster-b/auth/kubeconfig broker-info.subm --cluster-id cluster-b
 ```
 
-### Verify connectivity
+#### Verify connectivity
 
 This will run a series of E2E tests to verify proper connectivity between the cluster pods and services
 
 ```bash
 subctl verify-connectivity cluster-a/auth/kubeconfig cluster-b/auth/kubeconfig --verbose
 ```
+
+### Install Submariner with Service Discovery
```

Comment:
Let's make this a separate whole file, instead of modifying this quickstart.

We make it less confusing by having it separate, in the future we could merge into a single one when we get rid of kubefed and all the other quirks/risks.

Reply:
Done, created a new page.